src/index.js

```diff
@@ -29,6 +29,56 @@ Hypergrid.rectangular = require('rectangular');
 Hypergrid.lib = require('./lib');
 Hypergrid.Base = require('./Base');
 
+var windowRequire = window.require;
+
+window.require = function(path) {
+    var result, crumbs, i,
+        errMsg = 'Path ' + path + ' unknown or not exposed in build file.';
+
+    if (path.indexOf('fin-hypergrid/') === 0) {
```

Comment:
Yes I see that you using your require when the path is local to Hypergrid otherwise theirs. I still have concerns about this. Lets discuss after merge to master